api/helpers.ts

```diff
@@ -6,17 +6,17 @@
   convenience purposes.
 */
 
-import { createAsyncHandler } from "./utils.js";
 import type { Callback } from "../generator/interfaces";
-import type PingerV1 from "./facades/pinger-v1";
+import type PingerV1 from "./facades/pinger/PingerV1.js";
+import { createAsyncHandler } from "./utils.js";
 
 /**
   Decorate the Admin facade class.
 
   @param {Object} cls The auto-generated class.
   @returns {Object} The decorated class.
 */
-function wrapAdmin(cls) {
+function wrapAdmin(cls: any): object {
```

Comment:
Over time we can improve these `any`, `object`, `Function` etc. types.

Reply:
yes! I've a created an issue to track this: https://github.com/juju/js-libjuju/issues/93